### PR TITLE
Update babel cli

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Upgrades `babel-cli` to `@babel/cli`. Trickier than hoped because of Jest - see git commits for comments.

- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,6 +66,7 @@
     "react": "^16.6.3",
     "react-dom": "^16.6.3",
     "react-test-renderer": "^16.6.3",
+    "regenerator-runtime": "^0.13.1",
```
Comment (dr3) [inline package.json]: Whats this? I don't need it locally

Comment (ChrisBAshton) [reply package.json]: I think you'll find you do 🤔 I tried removing it and it errored, so I added it back in.

Needed by Jest.

**Review by radiocontrolled — APPROVED** [review]
👍

File: package.json (hunk 1 above)
Comment (dr3) [reply package.json]: Hmm running `npm run test` and `npm run build` I dont need them :/ Not sure why I dont and you do, strange

Comment (ChrisBAshton) [reply package.json]: Just double-checked - you're right! `jest` used to fail without it, but perhaps I was missing the env stuff in babel.config.js at that time.

Have now removed. Thanks!

**Review by dr3 — APPROVED** [review]

**Comment by ChrisBAshton** [comment]
For #94.

**Pull request merged**